### ISSUE TITLE
bug(mat-select): Screen reader announce selected option twice

### ISSUE THREAD

**Issue by angelbello87** [angular/components#26583]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

after selecting an option from the mat-select dropdown NVDA will announce the selection name twice
This happens with out of the box component or after adding the matSelectAccessibility class to the mat-select tag
![image](https://user-images.githubusercontent.com/14942214/217325939-1a502921-5b9d-4b21-9f45-a0e6246e2c3d.png)

### Reproduction

Steps to reproduce:
1. Go to https://material.angular.io/components/select/overview using Chrome or Edge
2. Open NVDA or Narrator
3. Navigate to the dropdown with the keyboard
4. Activate dropdown with the keyboard
5. Select an element from the dropdown with the keyboard

### Expected Behavior

It should announce the selected component only once

### Actual Behavior

It announces the selected component 2 times

### Environment

- Angular: 14.1.1
- CDK/Material: versions 13.2.3 and 15.1.3 
- Browser(s): Edge and Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by zarend** [comment]
Hello,

Thank you for reporting this issue. I wouldn’t expect this issue to block users from completing a workflow. For that reason, I’m updating the priority to P3. Please let us know if you have further concerns concerns and feel free to react with “👍” if you are impacted.

Best,
Zach

**Comment by angelbello87** [comment]
Hello Zach,
Thanks for the feedback. Just FYI I'm impacted 

Best,
Angel

**Comment by lm632678** [comment]
Hello Zach,
Does this issue fixed?

Best,
Lumin

**Comment by harish-byralingaiah** [comment]
Any update on the issue fix?

**Comment by zarend** [comment]
Hello, this is on my radar, but I haven't gotten to it yet. I have a fix in progress that requires me to test it cross browser with our supported screen readers. That way we know the fix is safe.

-Zach
